components/dashboard/public/schemas/gitpod-schema.json

```diff
@@ -5,16 +5,10 @@
     "type": "object",
     "properties": {
         "ide": {
-          "oneOf":[
```

Comment:
It does not work and breaks schema validation. I am not sure how allow any string or enum, for now it is just theia or code to provide content assist.

Reply:
it should be `anyOf`: https://github.com/json-schema-org/json-schema-spec/issues/665#issuecomment-432072391